android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -4947,6 +4947,17 @@ public void downloadFailed(@NonNull Exception e) {
         });
     }
 
+    public void setFormFieldHighlightColor(ReadableMap fieldHighlightColor) throws PDFNetException {
+        if (getPdfViewCtrl() != null) {
+            int red = fieldHighlightColor.getInt(COLOR_RED);
+            int green = fieldHighlightColor.getInt(COLOR_GREEN);
+            int blue = fieldHighlightColor.getInt(COLOR_BLUE);
+            getPdfViewCtrl().setFieldHighlightColor(new ColorPt(red/255., green/255., blue/255.));
```

Comment:
The `setFieldHighlightColor()` API is unusual in that it takes a 4-component `ColorPt` object, with the last component being the alpha.

(Normally a 4-component color is in the CMYK colorspace, not RGBA)

Reply:
fixed